client/modules/summary.js

```diff
@@ -0,0 +1,36 @@
+import {
+  defaultSummary
+} from './defaults';
+
+const REQUEST_SUM = 'budgeting-sample-app/summary/REQUEST_SUM';
+
+export const actions = {
+  requestSum: data => ({
+    type: REQUEST_SUM,
+    data
+  })
+};
+
+/**
+ * Summary calculation
+ * @param  {Object} state  Current state
+ * @param  {Object} action Dispatched action
+ * @return {Object}        Default state
+ */
+export default function reducer(state = defaultSummary, action) {
+  let sum;
+  switch (action.type) {
+    case REQUEST_SUM:
+      sum = action.data.reduce((prev, current) => (
+        { value: prev.value + current.value }
```

Comment:
Again, let's improve this in a separate commit.

Although the ducks spec says that a duck "MUST export default a function called reducer()" I can see your point. So, I renamed all of the reducers to <module>Reducer: transactionsReducer(), categoriesReducer() etc